torch/testing/_internal/common_methods_invocations.py

```diff
@@ -592,6 +595,8 @@ def __init__(self,
             else dtypesIfCUDA if dtypesIfCUDA is not None
             else dtypes)
 
+        if dtypesIfCPU is not None:
```

Comment:
Let's just kill this warning for now; I don't think people will use it but if they do that's fine